envelope.go

```diff
@@ -199,6 +199,31 @@ func parseBinaryOnlyBody(root *Part, e *Envelope) error {
 	return nil
 }
 
+func parseTextBody(p *Part) ([]byte, error) {
+
+	// use reader that part described
+	allBytes, ioerr := ioutil.ReadAll(p)
+	if ioerr != nil {
+
+		// quoted-printable was not quoted. try raw
+		if strings.Contains(ioerr.Error(), "quotedprintable: invalid unescaped byte") {
```

Comment:
In general, it's best to avoid inspecting to the contents of an error string. In this case I don't see an easy way around it though.

https://dave.cheney.net/2016/04/27/dont-just-check-errors-handle-them-gracefully (see Never inspect the output of error.Error)